assets/css/style.css

```diff
@@ -149,6 +151,48 @@ a {
 }
 
 
+
+/*Home Review Section*/
+.home-review-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+}
+.home-review-copy {
```

Comment:
Font face, weight, and size don't match the Figma designs for all view sizes. Please update.

Figma shows headings as Montserrat Alternates with the following sizes:
Desktop - 55 Bold
Tablet - 35 Bold
Mobile - 35 Bold

Reply:
I thought we had standard h2 settings set up since this was created so didn't want to create redundant css. I can add for now though

Reply:
I don't know what happened if someone had committed in my branch, but I was behind when I tried to push and lost some work. Will update by EOD.

Reply:
h3 header css has been added as well as some formatting adjustments

Reply:
I thought we had standard headings too but only found font-family set for all. Thanks for updating.